dataactvalidator/scripts/load_usps_files.py

```diff
@@ -325,15 +347,31 @@ def main():
 
     usps_file_dir = os.path.join(CONFIG_BROKER['path'], 'dataactvalidator', 'config', 'usps')
 
+    file_fulfilled_date, last_load_date_obj = None, None
+
     if args.download:
-        download_usps_files(usps_file_dir)
+        file_fulfilled_date, last_load_date_obj = download_usps_files(sess, usps_file_dir)
 
-    upload_files_to_s3(args, usps_file_dir)
+    # upload_files_to_s3(args, usps_file_dir)
 
     if args.remove:
         shutil.rmtree(usps_file_dir)
 
+    # update the last load date to match the fulfilled of the file that was just finished downloading
```

Comment:
I was thinking we would only update the date if we were able to successfully move the files to S3 so that if the script died during the S3 upload, the data wouldn't have to be reverted.

Reply:
Yes, that makes sense. The groovy pipeline runs through each argument individually in its own stage. We should update it to to run all of them at once (either by making adjusting this scripts arguments or updating the dev ops script).

Reply:
cc @kwhickey & @willkjackson to coordinate this change